.github/workflows/iroha2-pr-heavy.yml

```diff
@@ -1,72 +1,63 @@
-name: I2::Dev::Tests (Heavy)
+name: I2::Release::Tests
 
 on:
   pull_request:
     branches: [iroha2]
-    tags:
-      - "v*.*.*"
-    paths:
-      - "**.rs"
-      - "**.json"
-      - "**.toml"
-      - "**.yml"
-
-      # Not part of the workspace
-      - "!wasm/**"
 
 env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  test-docker:
-    runs-on: ubuntu-latest
+  cli-test:
+    runs-on: [self-hosted, linux]
     container:
       image: 7272721/i2-ci:latest
     steps:
       - uses: actions/checkout@v2
       - uses: Swatinem/rust-cache@v1
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v1
+      # - name: Set up Docker Buildx
+      #   id: buildx
+      #   uses: docker/setup-buildx-action@v1
       - name: Build Client CLI
         run: mold -run cargo build
         working-directory: client_cli
-      - name: Build and push Iroha Docker image
-        uses: docker/build-push-action@v2
-        with:
-          context: .
-          push: false
-          tags: hyperledger/iroha2:dev
-
-      - name: Setup docker test environment
-        run: ./scripts/setup_docker_test_env.sh
-      - name: Docker compose genesis test
-        run: bash -c './scripts/test_genesis_docker_compose.sh || ( docker-compose logs; false )'
-      - name: Docker compose test
-        run: bash -c './scripts/test_docker_compose.sh || ( docker-compose logs; false )'
-      - name: Cleanup docker test environment
-        run: ./scripts/cleanup_docker_test_env.sh
+      # - name: Build and push Iroha Docker image
+      #   uses: docker/build-push-action@v2
+      #   with:
+      #     context: .
+      #     push: false
+      #     tags: hyperledger/iroha2:dev
+      - name: Build Iroha Peer
+        run: mold -run cargo build
+        working-directory: cli
+      - name: Mark both binaries as executable
+        run: chmod +x target/debug/iroha && chmod +x target/debug/iroha_client_cli
```

Comment:
how come they are not marked as executable?

Reply:
I think paranoid security policies on the host filesystem. but I don't know for sure.